packages/cli/test/mocks/client.ts

```diff
@@ -29,6 +36,40 @@ class MockStream extends PassThrough {
   // These are for the `ora` module
   clearLine() {}
   cursorTo() {}
+
+  override _write(
+    chunk: any,
+    encoding: BufferEncoding,
+    callback: (error?: Error | null | undefined) => void
+  ): void {
+    const str = chunk.toString();
+
+    this.#_fullOutput += str;
+
+    // There's some ANSI Inquirer just send to keep state of the terminal clear; we'll ignore those since they're
+    // unlikely to be used by end users or part of prompt code.
+    if (!ignoredAnsi.has(str)) {
+      this.#_rawChunks.push(str);
+    }
+
+    // Stripping the ANSI codes here because Inquirer will push commands ANSI (like cursor move.)
+    // This is probably fine since we don't care about those for testing; but this could become
+    // an issue if we ever want to test for those.
+    if (stripAnsi(str).trim().length > 0) {
+      this.#_chunks.push(str);
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏should this push the stripped version instead of the original `str`?

Reply:
I believe the idea here is to determine _if_ we should log this string. As in, if we remove any ansi sequences from the string, is there anything to show? If so, add it. And then the actual stripping is done in the `getScreen` call.

For context, this is copied from inquirer test https://github.com/SBoudrias/Inquirer.js/blob/master/packages/testing/src/index.mts

Reply:
Ohh, got it